docs/usage/00-installing.adoc

```diff
@@ -27,7 +27,7 @@ To install `dklb`, the following prerequisites must be met:
 ** DC/OS must be v1.12.0 or later.
 ** EdgeLB must be built and installed from the https://github.com/mesosphere/dcos-edge-lb/tree/master[`master`] branch.
    A stub-universe is available, see https://github.com/mesosphere/dcos-edge-lb#stub-universe[here] for details.
-*** EdgeLB https://docs.mesosphere.com/services/edge-lb/1.3/[`v1.3.0`] may be used instead, but some functionality will be missing.
+*** EdgeLB https://docs.mesosphere.com/services/edge-lb/1.4/[`v1.4.0`] may be used instead, but some functionality will be missing.
```

Comment:
Is there functionality missing in 1.4.0? Why use it then?

Reply:
I think the original intent was to say that you should use `master`. What do you think of changing to `but some functionality might be missing`?